### DISCUSSION TITLE
Does anyone also wait for WebGPU

### DISCUSSION THREAD

**Discussion by fromwhite** [visgl/deck.gl#8041] [Q&A] [answered]
Will the next version update be Luma? 
Is the release v9 version WebGPU available?

**Comment by ibgreen** [comment, marked as answer]
@fromwhite Yes, deck.gl v9 will be based on luma.gl v9 which is WebGPU-enabled. Work is underway, but the effort is very extensive since WebGPU is a completely new API that does not make any efforts to be compatible with WebGL. 

While our high level ambition is to launch this year, At this time. it is not possible to give a firm timeline. There are a number of tasks that could be done by the community if there was interest in helping us accelerate the launch.

**Comment by birkskyum** [comment]
more updated info on deck.gl v9 status can be found in here:
 - https://github.com/visgl/deck.gl/issues/7457
